app/decorators/product_decorator.rb

```diff
@@ -79,7 +79,13 @@ def markings
     object.markings.join(", ")
   end
 
-  def to_csv
-    attributes.keys.map { |key| send(key) }
+  def case_ids
+    object.investigations.map(&:pretty_id).to_s
```

Comment:
```suggestion
    object.investigations.map(&:pretty_id)
```
CSV should do this automatically.